doc/interfaces/reconstruction_flow.rst

```diff
@@ -0,0 +1,269 @@
+.. _reconstruction_flow:
+
+==============
+Reconstruction
+==============
+
+This tutorial walks through the steps to perform reconstruction using DIPY.
+Multiple reconstruction methods are available in DIPY.
+
+You can try these methods using your own data; we will be using the data in DIPY.
+You can check how to :ref:`fetch the DIPY data<data_fetch>`.
+
+-----------------------------------
+Constrained Spherical Deconvolution
+-----------------------------------
+
+This method is mainly useful with datasets with gradient directions acquired on a
+spherical grid.
```

Comment:
I think datasets acquired in Cartesian coordinates can be resampled to spherical coordinates so as to use SD methods.

Reply:
Addressed in 4ce067c